usecase/usecase-webapi-xwalk-tests/samples/IAPGooglePlay/REAME.md

```diff
@@ -0,0 +1,11 @@
+# In-App Purchase Example for Google Play
+## Introduction
+This example is for testing the in-app purchase API for Google Play.
+## Pre-condition
+* Get the latest ipa.zip from [Crosswalk android extension iap example](https://github.com/crosswalk-project/crosswalk-android-extensions/releases/), unzip it and copy the unzipped files to IAPGooglePlay/iap directory.
+* Make sure Google Play and Google Wallet have been installed in the Android system.
+* The Google Wallet should bind with a credit card.
+* The Google account you logined in has been added to the test group of this app.
```

Comment:
"The Google Wallet should bind with a credit card.The Google account you logined in has been added to the test group of this app." -> "A Google account with an available Google wallet which should been granted as a tester of the test app com.crosswalk.iapsample from google play."